libs/ts-rest/serverless/src/aws.ts

```diff
@@ -0,0 +1 @@
+export * from './lib/handlers/ts-rest-lambda';
```

Comment:
I'm confused about how consumers will implement this? Will it be a plugin? Will it be also implemented at the handler level (allow users to import utility types/functions for their handlers)? 

Is it possible to generalize this a little further such that something like SST could also use this?

Reply:
It just creates a handler function that can be exported, you can see see some examples in ts-rest-lambda.spec.ts. You can already use this with SST 😉

```typescript
import { createLambdaHandler } from '@ts-rest/serverless/aws';

export const handler = createLambdaHandler(contract, { ... });
```

This creates a handler function that accepts a lambda event and context as arguments `function (event, context)`, which is standard for all AWS lambdas. This is as generic as it gets for AWS Lambdas. If there is a layer on top like SST that defines the handlers in a different format, you can just call that handler function manually and make sure you pass the lambda event and context from wherever it's defined for that specific "framework". So something like this:

```typescript
import { createLambdaHandler } from '@ts-rest/serverless/aws';

const tsRestHandler = createLambdaHandler(contract, { ... });

export const customHandler = async (nonStandardOptions) => {
  return tsRestHandler(nonStandardOptions.event, nonStandardOptions.context);
};
```

Reply:
SST just uses the standard `Handler` type from `aws-lambda`. We're using `createLambdaHandler` directly in our applications. Works well.

Reply:
can you give more example of SST

Reply:
kindly help